docker-run-eltern-emailer.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
Should we update the instructions in text.md to explain how to use this with Docker?

Reply:
good idea. I added instructions and removed that file, because IMHO users will use their own command line.